graviton/blackbox.py

```diff
@@ -46,222 +35,6 @@
 MAX_APP_ARG_LIMIT = atc.AtomicTransactionComposer.MAX_APP_ARG_LIMIT
 
 
-class DryRunProperty(Enum):
```

Comment:
This, and several other non-execution related code has been broken out to `Inspector.py`

Reply:
much better